app/models/mixins/authentication_mixin.rb

```diff
@@ -79,14 +79,7 @@ def authentication_service_account(type = nil)
   end
 
   def required_credential_fields(type)
-    case type.to_s
-    when "bearer"
-      [:auth_key]
-    when "hawkular"
-      []
-    else
-      [:userid]
-    end
+    []
```

Comment:
@simon3z 
> Is the test you're adding demonstrating that we don't need this in any case?

:+1:  Added a test to check that now we can not create an authentication type `:hawkular` without the `:auth_key` field, The test in the original version of this PR did not check that.

The new test will fail without the fix in this PR.